frappe/public/js/frappe/views/kanban/kanban_view.js

```diff
@@ -107,13 +107,7 @@ frappe.views.KanbanView = class KanbanView extends frappe.views.ListView {
 	save_kanban_board_filters() {
 		const filters = this.filter_area.get();
 
-		frappe.call({
-			method: 'frappe.desk.doctype.kanban_board.kanban_board.save_filters',
-			args: {
-				board_name: this.board_name,
-				filters: filters
-			}
-		}).then(r => {
+		frappe.db.set_value("Kanban Board", this.board_name, "filters", filters).then(r => {
```

Comment:
@gavindsouza I guess this will cause permission issue for users who do not have write access to Kanban Board doctype. Please verify.

Reply:
That was the desired effect. People who don't have write access shouldn't be able to set filters that affect other users' experience with the respective Kanban board.